services/eth_gateway.go

```diff
@@ -143,6 +147,21 @@ func generateEthAddr() (addr common.Address, privateKey string, err error) {
 	return addr, privateKey, err
 }
 
+// Generate an array of eth keys
+func generateKeys(numKeys int) []string {
+	var keys []string
+	for i := 0; i < numKeys; i++ {
+		key := ""
+		if oyster_utils.BrokerMode == oyster_utils.TestModeDummyTreasure {
+			key = os.Getenv("TEST_MODE_WALLET_KEY")
+		} else {
+			_, key, _ = generateEthAddr()
```

Comment:
Do you need to check error, if there is something wrong, you should not add the key into keys

Reply:
I'll add it